analytical/templatetags/uservoice.py

```diff
@@ -56,8 +56,8 @@ def render(self, context):
         widget_key = context.get('uservoice_widget_key')
         if not widget_key:
             widget_key = self.default_widget_key
```

Comment:
This could probably be written as:
```python
widget_key = context.get('uservoice_widget_key') or self.default_widget_key
```
Then there would only be a need for a single `if` subsequently, which might be less confusing.